autogen/agentchat/conversable_agent.py

```diff
@@ -638,10 +672,12 @@ def generate_oai_reply(
             context=messages[-1].pop("context", None), messages=self._oai_system_message + messages
         )
 
-        # TODO: line 301, line 271 is converting messages to dict. Can be removed after ChatCompletionMessage_to_dict is merged.
+        # TODO: line 270, 297, 431 are converting messages to dict. Can be removed after ChatCompletionMessage_to_dict is merged.
```

Comment:
it looks like https://github.com/microsoft/autogen/pull/791 has already been merged. Should we remove this TODO?

Reply:
That's referencing the message_to_dict method, looks like it is actually also used by groupchat and may have some other complications, so probably better in another pr

Reply:
In that case let's remove this comment because it is going to confuse people.